src/types/posts.ts

```diff
@@ -3,6 +3,7 @@ export interface PostResponse {
 }
 
 export interface Post {
+    [x: string]: any;
```

Comment:
This doesn't sound right. Why is this necessary?

This type of typing makes sense when you have vague information of structure. For example, we have a page that requires a user to sign in. For some users, an endpoint returns an object that has 10 items. For others, it has 20. For admin, it could go all the way to 200. You don't want to write all the possibilities (or you might, depends on situation).

Posts in our subreddit are specific. They have a specific list of properties - at least according to apiary. So why is this necessary?

Reply:
It was added by accident when I was choosing from the options offered by ctrl+.